doc/use-cases-old-module.md

```diff
@@ -0,0 +1,351 @@
+# Use-cases of the Old YaST Users Module
```

Comment:
This mentions the old yast module assuming a new one is to be created. That's not the case anymore.